yellowbrick/classifier/classification_report.py

```diff
@@ -129,16 +138,21 @@ def score(self, X, y=None, **kwargs):
         scores = list(scores)
         scores[-1] = scores[-1] / scores[-1].sum()
 
-        # Create a mapping composed of precision,recall, presion, and support
+        # Create a mapping composed of precision, recall, F1, and support
         # to their respective values
         scores = map(lambda s: dict(zip(self.classes_, s)), scores)
-        self.scores_ = dict(zip(SCORES_KEYS, scores))
+        self.scores_map_ = dict(zip(SCORES_KEYS, scores))
```

Comment:
This is very minor, though I've probably given it more thought than I should have. I like the change in `ClassBalance` to store what used to be `scores_` into a `predictions_` table. Keeping the `score_` property more consistent as a single value metric throughout the `ClassificationScoreVisualizer` subclasses. 

However, here I would ask that we maintain this property as `self.scores_`. Although this might create a bit of confusion between `score_` and `scores_`, I believe it is important here to maintain backward compatibility with folks who were looking to fetch deeper metrics of this class (I know I have quite a bit of code that uses this attribute).

Reply:
Reverted